man/ctags.1.rst.in

```diff
@@ -549,6 +549,12 @@ See "`TAG ENTRIES`_" about fields, kinds, roles, and extras.
 		to which it is applied is not subject to change. Selecting this
 		option type causes the following options to be ignored: ``-B``, ``-F``.
 
+		``number`` type is ignored in Xref and JSON output formats. Use
+		``--_xformat="...%n"`` for Xref output format, or ``--fields=+n`` for
```

Comment:
Maybe ``--fields=+n-P`` is a bit better than ``--fields=+n`` because `-P` suppresses `"pattern:"` field of JSON.
 
I didn't know we can turn off the pattern field in JSON mode but it was tested in fixed-field-handling.d :-O

Could you add the following change as complements:
```diff
diff --git a/man/ctags.1.rst.in b/man/ctags.1.rst.in
index 2fee7089e..b010de714 100644
--- a/man/ctags.1.rst.in
+++ b/man/ctags.1.rst.in
@@ -1080,10 +1080,13 @@ Listing Options
 		``--output-format=json`` are still experimental.
 
 	FIXED
-	   Whether this field can be disabled or not.
+	   Whether this field can be disabled or not in tags output.
 
-	   Some fields are printed always
-	   in tags output. They have ``yes`` as the value for this column.
+	   Some fields are printed always in tags output.
+	   They have ``yes`` as the value for this column.
+
+	   Unlike the tag output mode, json output mode allows disabling
+	   any fields.
 
 	DESCRIPTION
 		Human readable description for the field.

```

Reply:
```
$ ctags --output-format=json --fields=+n-P -o - qualified.java
{"_type": "tag", "name": "point", "path": "qualified.java", "line": 1, "kind": "class"}
{"_type": "tag", "name": "x", "path": "qualified.java", "line": 2, "kind": "field", "scope": "point", "scopeKind": "class"}
$ ctags --fields=+n-P --output-format=json -o - qualified.java
ctags: Warning: Cannot disable fixed field: 'P'{pattern}
{"_type": "tag", "name": "point", "path": "qualified.java", "pattern": "/^class point {$/", "line": 1, "kind": "class"}
{"_type": "tag", "name": "x", "path": "qualified.java", "pattern": "/^\tdouble x;$/", "line": 2, "kind": "field", "scope": "point", "scopeKind": "class"}
$
```

We need document about this.

Reply:
I've commit the fix you proposed.

Reply:
I have tried to fix the issue about the order of options in #2861.

Reply:
You removed the argument `bool warnIfFixedField` from `enableField()` and still warns if a field is fixed.
I will delete the commit for this issue.